### PR TITLE
Implement the blockchain application removal flow- Closes #4375 

### LINKED ISSUE #4375: Implement blockchain application removal
### Description
User can remove an application from the local storage by visiting application management
User can not remove the default application provided by lisk foundation
In case user remove the current application it should be navigate to lisk application	

### Acceptance Criteria

- Step 1: Allow user to review the application details Ask for confirmation 
- Allow users to show application details such as Application name, avatar, owner address, application status, chainId. Last update, Last Certificate Height, deposit, application URL, custom background
- Step 2: Show user the success page and back to the dashboard`
- Check for the isDefault application to avoid default application remove required [design](https://www.figma.com/file/uStjr4eQ37wPm9bDt8JyXP/Application-exploring-and-management?node-id=136%3A9869)
- Ensure [UI/UX design](https://www.figma.com/file/uStjr4eQ37wPm9bDt8JyXP/Application-exploring-and-management?node-id=136%3A10371) requirements are met
- Add required unit tests
- Add required new e2e tests

	
### Additional Information

- Use the ‘useApplicationsManagement’ hook to remove the application
- Suggestion route `application/remove?chainId=XXXX`
- Suggestion component name `RemoveApplicationFlow`
- Suggestion component name `RemoveApplicationSuccess`
- Suggestion component name `RemoveApplicationDetails`

Blocked by #4371

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #4375 

### How was it solved?

- [x] Implement the `RemoveApplicationDetails` component
- [x] Implement the `RemoveApplicationSuccess` component
- [x] integrate `RemoveApplicationDetails`, `RemoveApplicationSuccess` and `useApplicationManagement` hook to create the Whole flow
- [x] implement unit test
- [x] implement e2e tests

NOTE:  Some e2e tests could not be achieved in this pr because presently we can't set the current application on the app on runtime. but after #4359 is done then these tests can be achieved

### How was it tested?

jest
e2e

### CODE REVIEW

File: tests/cypress/features/blockchainManagement.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: BlockchainManagement
+  @basic
+  Scenario: Remove blockchain application
+    Given I visit the remove blockchain application page
+    And I wait 1 seconds
+    Then blockchain details should be accuratly displayed
```
Comment (ikem-legend) [inline tests/cypress/features/blockchainManagement.feature]: @eniolam1000752 Minor typo for `accuratly`

File: tests/cypress/features/blockchainManagement/blockchainManagement.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import mockBlockchainApplications from '../../../fixtures/blockchainApplicationsExplore';
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
+
+const blockchainApplication= mockBlockchainApplications[0]
+
+Given(/^I visit the remove blockchain application page$/, function () {
+  cy.visit(`${urls.login}?modal=removeApplicationFlow&chainId=${blockchainApplication.chainID}`);
+});
+
+Then(/^blockchain details should be accuratly displayed$/, function () {
```
Comment (ikem-legend) [inline tests/cypress/features/blockchainManagement/blockchainManagement.js]: @eniolam1000752 Previous typo would also be fixed here

**Review by soroushm — DISMISSED** [review]

File: src/modules/blockchainApplication/manage/components/RemoveApplicationDetails/RemoveApplicationDetails.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    },
+    {
+      header: t('Last Certificate Height'),
+      className: `${styles.detailContentText} last-certificate-height`,
+      content: lastCertificateHeight,
+    },
+  ];
+
+  const handleRemoveApplication = () => {
+    deleteApplicationByChainId(chainID);
```
Comment (ikem-legend) [inline src/modules/blockchainApplication/manage/components/RemoveApplicationDetails/RemoveApplicationDetails.js]: @eniolam1000752 Please add TODO as regards the issue with this functionality

File: src/modules/blockchainApplication/manage/components/RemoveApplicationDetails/RemoveApplicationDetails.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+            </ValueAndLabel>
+          </Box>
+          <div className={styles.addressRow}>
+            <Link
+              className={`${styles.appLink}`}
+              target="_blank"
+                // eslint-disable-next-line
+                // TODO: this is just a place holder link pending when its part of the response payload from service
+              to={serviceUrl}
+            >
```
Comment (ikem-legend) [inline src/modules/blockchainApplication/manage/components/RemoveApplicationDetails/RemoveApplicationDetails.js]: @eniolam1000752 Please replace this with anchor tag to ensure that it works appropriately

**Review by ikem-legend — APPROVED** [review]

**Review by soroushm — APPROVED** [review]

**Pull request merged**